src/App.js

```diff
@@ -1,16 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatMessages);
+
+  const toggleLikeButton = (id) => {
+    setChatData(chatData => chatData.map(chat => {
+      if (chat.id === id) {
+        return {...chat, liked: !chat.liked}
+      } else {
+        return chat;
+      }
+    }))
+  };
+
+  const calcTotalLikes = (chatData) => {
+    let everyHeart = '❤️'
+    let numberOfHearts = 0
+    return chatData.reduce((likeTotal, chat) => {
+      if (chat.liked) {
+        numberOfHearts += 1
+      }
+      return `${numberOfHearts}${everyHeart}`;
+    }, 1);
+  };
+
+  const displayTotalLikes = calcTotalLikes(chatData);
+
   return (
     <div id="App">
-      <header>
-        <h1>Application title</h1>
+      <header className="App-header">
+        <h1>Ghibli Jabber</h1>
+        <section>
+          <h2 className="likes-counter" >Likes: {displayTotalLikes}</h2>
```

Comment:
Following up on the earlier comment, I mean you can do this if you follow the reduce pattern I laid out:
```
<h2 className="likes-counter" >Likes: {displayTotalLikes} ❤️s</h2>
```
Which will help your tests to pass as well.